Update spawning.md (#674)
tokio-rs/website: content/tokio/tutorial/spawning.md

When we say that a value is `'static`, all that means is that it would not be
incorrect to keep that value around forever. This is important because the
compiler is unable to reason about how long a newly spawned task stays around.

> We have to make sure that the task is allowed to live forever, so that Tokio can make the task run as long as it needs to.

Reviewer: ```suggestion
We have to make sure that the task is allowed to live forever, so that Tokio
can make the task run as long as it needs to.
```

When we say that a value is `'static`, all that means is that it would not be
incorrect to keep that value around forever. This is important because the
compiler is unable to reason about how long a newly spawned task stays around.

> We have to make sure the task is allowed to live forever, in order to let the compiler
> give the permission to tokio to make the task run as long as it needs to.

Reviewer: ```suggestion
We have to make sure that the task is allowed to live forever, so that Tokio can make the task run as long as it needs to.
```

Author: That's more concise ! It does not mention the compiler needing to check the lifetime of the task, but since it's mentioned above, I think it's clear like that :)
